omero/developers/Tables.rst

```diff
@@ -301,6 +301,99 @@ Examples
    :source:`examples/OmeroTables/FindMeasurements.java <examples/OmeroTables/FindMeasurements.java>`
 
 
+Data display in OMERO.web
+^^^^^^^^^^^^^^^^^^^^^^^^^
+The following code creates a table for a dataset that contains a few images.
+
+.. code-block:: python
+
+  from getpass import getpass
+  from random import random
+  from omero.gateway import BlitzGateway
+  from omero.grid import LongColumn, StringColumn, DoubleColumn, ImageColumn
+  from omero.model import FileAnnotationI, OriginalFileI, ImageAnnotationLinkI, DatasetAnnotationLinkI, ImageI, DatasetI
+
+
+  def create_table(connection, dataset_id):
+      dataset = connection.getObject('Dataset', dataset_id)
+      images = [connection.getObject('Image', child.getId()) for child in dataset.listChildren()]
+
+      # Create some data
+      data = []
+      for image in images:
+          data.append(
+              {
+                  'image': image.getId(),
```

Comment:
It's kinda minor point but the convention we've adopted now is to use `Image` rather than `image` as the column name. E.g. see https://github.com/ome/omero-metadata#populate. I realise we've used `image` in some of the older IDR tables (and it certainly works), but more recent ones (created by the current `omero-metadata` tool) use `Image`.